sqlglot/dialects/clickzetta.py

```diff
@@ -160,6 +163,58 @@ def adjust_day_of_week(self: ClickZetta.Generator, expression: exp.DayOfWeek):
     else:
         return f'DAYOFWEEK({self.sql(expression.this)})'
 
+
+def _group_sql(self: ClickZetta.Generator, expression: exp.Group):
```

Comment:
Maybe transform.preprocess is a better and more light weight way here, no need to re-implement whole group_sql func.